doc/lightweight-styling.md

```diff
@@ -130,6 +130,74 @@ With this XAML we are given the following visual result, notice the third Button
 
 ![Material - Button lightweight styling](assets/material-button-pointerover-lightweight-styling.png)
 
+## C# Markup
+
+All Lightweight Styling resource key can also be used in C# Markup through a collection of static helper classes available in the [Uno.Themes.WinUI.Markup](https://www.nuget.org/packages/Uno.Themes.WinUI.Markup/) NuGet package. The following code shows how to override several `FilledButton` resources in C# from the previous XAML example above. Notice that the `Button` is still using the `FilledButtonStyle` from Uno Material, but the resources are being overridden.
+
+```csharp
+// #1
+new Button()
+    .Style(Theme.Button.Styles.Filled)
+    .Content("Default Button Style"),
+
+//#2
+new Button()
+    .Style(Theme.Button.Styles.Filled)
+    .Resources(config => config
+        .Add(Theme.Button.Resources.BorderThickness, 2)
+        .Add(Theme.Button.Resources.Filled.Foreground.Default, new SolidColorBrush(Colors.DarkGreen))
+        .Add(Theme.Button.Resources.Filled.Background.Default, new SolidColorBrush(Colors.LightGreen))
+        .Add(Theme.Button.Resources.Filled.BorderBrush.Default, new SolidColorBrush(Colors.DarkGreen))
+    )
+    .Content("Overridden Button Style"),
+
+//#3
+new Button()
```

Comment:
avoid what could trigger issue linking:
```suggestion
// basic filled button
new Button()
    .Style(Theme.Button.Styles.Filled)
    .Content("Default Button Style"),

// filled button with overridden colors
new Button()
    .Style(Theme.Button.Styles.Filled)
    .Resources(config => config
        .Add(Theme.Button.Resources.BorderThickness, 2)
        .Add(Theme.Button.Resources.Filled.Foreground.Default, new SolidColorBrush(Colors.DarkGreen))
        .Add(Theme.Button.Resources.Filled.Background.Default, new SolidColorBrush(Colors.LightGreen))
        .Add(Theme.Button.Resources.Filled.BorderBrush.Default, new SolidColorBrush(Colors.DarkGreen))
    )
    .Content("Overridden Button Style"),

// filled button with overridden colors for PointerOver state
new Button()
```

Reply:
Comments are to match the <!-- --> comments from the previous XAML code

Reply:
Oh I see the problem, ok will change